app/helpers/application_helper.rb

```diff
@@ -5,11 +5,11 @@ module ApplicationHelper
   include Spotlight::ApplicationHelper
   delegate :url, to: :universal_viewer, prefix: true
 
-  def render_search_bar
-    super
-  rescue StandardError
-    render partial: 'catalog/default_search_form'
-  end
+  # def render_search_bar
+  #   super
+  # rescue StandardError
+  #   render partial: 'catalog/default_search_form'
+  # end
```

Comment:
question: Is this no longer needed?

Reply:
@eliotjordan I don't *think* this is needed, and it would be simpler to use the default, but I'm not sure why we were overriding the default searchbox if it looks the same.